azure-pipelines.yaml

```diff
@@ -50,7 +50,7 @@ stages:
         steps:
           - bash: ./src/qa-test-eks.sh init
             displayName: "Init"
-          - bash: ./src/qa-test-eks.sh cleanup qa19
+          - bash: ./src/qa-test-eks.sh cleanup eu-west-1 qa19
             displayName: "Pre-apply cleanup"
```

Comment:
I think it's useful if there's a resource left over from a previous run. You can re-run the pipeline with the variable set and it will clean it up (provided it is found by the query of course). The alternative is to do a manual clean up?

Reply:
Sounds reasonable.